plugin/src/main/groovy/com/novoda/staticanalysis/internal/ktlint/KtlintConfigurator.groovy

```diff
@@ -9,11 +9,13 @@ import org.gradle.api.GradleException
 import org.gradle.api.NamedDomainObjectContainer
 import org.gradle.api.Project
 import org.gradle.api.Task
+import org.gradle.api.file.RegularFileProperty
```

Comment:
Is this putting a cap on the minimum version of Gradle we support now?

Reply:
I can see from the javadocs that this is since Gradle 4.3. I don't think this is a blocker per-se, just never thought of the implicit compatibility with Gradle API until now.

Reply:
Hmmm, since ktlint requires Gradle 4.3 anyways, it might not be a problem. I wonder what happens with the other tools when Gradle is old and ktlint is not used. I will check that.

Reply:
Probably not worth the hassle right now. We might have already an implicit dependency we never realised before. Also worst case scenario this will crash on projects running on Gradle < 4.3 only at runtime (and as you said KtLint will need 4.3+ anyway).

Reply:
If you're sure that it is a runtime crash, then I think we are good to go with that. :+1:

Reply:
I'm like 99.9% sure that will be a crash at configuration time - the same it will happen when trying to use KtLint with older versions of Gradle

Reply:
It will crash at configuration only when KtLint is used right?

Reply:
I tried it but not so easy in a single project. Will try once we get a snapshot.